.github/workflows/check-pr-title.yml

```diff
@@ -0,0 +1,22 @@
+# NOTE: This is a common file that is overwritten by realm/ci-actions sync service
+# and should only be modified in that repository.
+
+name: "Check PR Title"
+on:
+  pull_request:
+    types: [opened, synchronize, reopened, ready_for_review, labeled, unlabeled, converted_to_draft, edited]
+
+jobs:
+  check-pr-title:
+    name: Check PR Title
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v4
+        with:
+          submodules: false
+      - name: Enforce PR title
+        uses: realm/ci-actions/title-checker@main
+        with:
+          regex: R[A-Z]{2,5}-[0-9]{1,6}
```

Comment:
This does not seem to match `RKOTLIN` 🤔

Reply:
Ugh, I thought RSWIFT was the longest one 🤦‍♂️